src/server/dfly_main.cc

```diff
@@ -339,6 +344,16 @@ Usage: dragonfly [FLAGS]
   mi_option_enable(mi_option_show_errors);
   mi_option_set(mi_option_max_warnings, 0);
 
+  const auto& cluster_mode = GetFlag(FLAGS_cluster_mode);
```

Comment:
define it as `string cluster_mode = ... `, also the regular comparison `==` will do the job just fine (no need for `iequals`).